packages/odyssey-design-tokens/src/border.json

```diff
@@ -16,7 +16,7 @@
     },
     "radius": {
       "main": { "value": "4px" },
-      "outer": { "value": "6px" },
+      "outer": { "value": "12px" },
       "round": { "value": "1.5em" }
```

Comment:
I really think these should be `rem` values. Am I wrong in thinking that? What's the difference between px and rem here? As far as I understand it, if you have a higher DPI, it will be an increasingly smaller and smaller radius than you'd expect, so the buttons will look different at higher DPI.

Reply:
For now, Design doesn't want the border radii to scale with the font size. The exception here is "round," which needs to scale to the local size to have that pill shape.